lib/LedgerSMB/Scripts/setup.pm

```diff
@@ -792,38 +792,43 @@ verify_check => md5_hex($check->test_query),
     my @skip_keys = grep /^skip_/, keys %$request;
     $hiddens->{@skip_keys} = $request->{@skip_keys};
 
+    my $cols = [];
+    for my $column (@{$check->columns // []}) {
```

Comment:
display_cols are to be displayed. columns holds the editable values.

Reply:
Also, shouldn't that be `for my $column (@{$check->columns} // []) {`

Reply:
Nope, should not.

Reply:
> display_cols are to be displayed. columns holds the editable values.
Working on a change which gets this right, right now.